src/Nethermind/Nethermind.Blockchain/BaseLoopBlockProducer.cs

```diff
@@ -25,9 +26,12 @@ namespace Nethermind.Blockchain
 {
     public abstract class BaseLoopBlockProducer : BaseBlockProducer
     {
+        private const int ChainNotYetProcessedMillisecondsDelay = 100;
         private readonly string _name;
```

Comment:
some blocks can take a second or longer to process

Reply:
It will just wait 100 ms multiple times